students/jungchan/users/views.py

```diff
@@ -1,3 +1,39 @@
 from django.shortcuts import render
 
 # Create your views here.
+import json, re
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import UserInformation
+
+class SignupView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if not data['email'] or not data['password']:
+                return JsonResponse({'message':'KEY_ERROR'}, status=400)
+            # email과 password가 입력되지 않았을 경우 띄워주는 경고
+            if re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', data['email']) == None:
+                return JsonResponse({'message':'Invalid_email'}, status=400)
```

Comment:
```suggestion
            if not re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', data['email']):
                return JsonResponse({'message':'Invalid_email'}, status=400)
```
python에서 None을 비교할 때느 == 을 잘 사용하지 않습니다. None 자체가 False의 의미를 가지고 있기 때문입니다. 
따라서 None일 때 조건문을 실행하고 싶은면 앞에 not을 붙혀주시면 됩니다.

아래 다른 if들도 수정해 주세요.